addons/binding/org.openhab.binding.squeezebox/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,243 @@
+	</channel-type>
+	<channel-type id="currentPlaylistShuffle">
+		<item-type>Number</item-type>
+		<label>Shuffle Mode</label>
+		<description>Current playlist shuffle mode</description>
+				<state>
+			<options>
+				<option value="0">No Shuffle</option>
+				<option value="1">Shuffle Songs in PLaylist</option>
+				<option value="2">Shuffle Albums in PLaylist</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="currentPlaylistRepeat">
+		<item-type>Number</item-type>
+		<label>Repeat Mode</label>
+		<description>Current playlist repeat Mode</description>
+		<state>
+			<options>
+				<option value="0">No Repeat</option>
+				<option value="1">Repeat Song</option>
+				<option value="2">Repeat Playlist</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<!-- Squeezebox variables -->
+	<channel-type id="title">
+		<item-type>String</item-type>
+		<label>Title</label>
+		<description>Title of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="remotetitle">
+		<item-type>String</item-type>
+		<label>Remote Title</label>
+		<description>Remote Title (Radio) of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="album">
+		<item-type>String</item-type>
+		<label>Album</label>
+		<description>Album name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="artist">
+		<item-type>String</item-type>
+		<label>Artist</label>
+		<description>Artist name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="year">
+		<item-type>String</item-type>
+		<label>Year</label>
+		<description>Release year of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="genre">
+		<item-type>String</item-type>
+		<label>Genre</label>
+		<description>Genre name of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="coverarturl">
+		<item-type>String</item-type>
+		<label>Cover Art</label>
+		<description>Address to cover art of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="coverartdata">
+		<item-type>Image</item-type>
+		<label>Cover Art</label>
+		<description>Image data of cover art of the current song</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ircode">
+		<item-type>String</item-type>
+		<label>IR Code</label>
+		<description>String of the cached IR code</description>
+		<state readOnly="true"></state>
+	</channel-type>
+		<channel-type id="numberPlaylistTracks">
+		<item-type>Number</item-type>
+		<label>Number of Playlist Tracks</label>
+		<description>Number of playlist tracks</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<!-- Advanced Player Properties -->
+	<channel-type id="ip" advanced="true">
```

Comment:
I think it should be a property (now that i know items can have properties) , the IP address is informational only, the MAC is used to address players and is configurable in the binding.